misc/multiaddr/src/protocol.rs

```diff
@@ -15,6 +15,8 @@ use std::{
 };
 use unsigned_varint::{encode, decode};
 
+pub use crate::onion_addr::Onion3Addr;
```

Comment:
In here you can just `use crate::onion_addr::Onion3Addr`, the `pub use` must go into `lib.rs` for `Onion3Addr` to be exported. The `onion_addr` module itself is private.